src/main/java/com/xceptance/xlt/mastercontroller/Main.java

```diff
@@ -201,7 +201,7 @@ private static HessianProxyFactory getHessianProxyFactory(final MasterController
      *            master controller configuration
      * @return the UrlConnectionFactory
      */
-    private static UrlConnectionFactory getUrlConnectionFactory(final MasterControllerConfiguration config)
+    private UrlConnectionFactory getUrlConnectionFactory(final MasterControllerConfiguration config)
```

Comment:
Shouldn't this be protected to allow overrides?

Reply:
Done.